doc-content/enterprise-only/openshift/additional-server-managed-deploy-assy.adoc

```diff
@@ -9,6 +9,8 @@ ifdef::context[:parent-context: {context}]
 
 You can deploy an additional managed {KIE_SERVER} to an authoring or managed environment.  Deploy the server in the same project as the {LOCAL_CENTRAL} deployment.
 
+In order for the {LOCAL_CENTRAL} to connect to the {KIE_SERVER}, you must enable the `OpenShiftStartupStrategy` setting for your environment, as described in <<startupstrategy-enable-proc>>.
```

Comment:
Suggest the following minor rewrite:

You must enable the `OpenShiftStartupStrategy` setting for your environment for the {LOCAL_CENTRAL} to connect to the {KIE_SERVER} as described in <<startupstrategy-enable-proc>>.